src/ol/interaction/modify.js

```diff
@@ -184,6 +184,10 @@ ol.interaction.Modify = function(options) {
    */
   this.lastPointerEvent_ = null;
 
+  ol.events.listen(this,
```

Comment:
Wouldn't it be better to override the `setActive()` method, instead of registering a listener here?